shell/packages/sandstorm-db/db.js

```diff
@@ -112,6 +112,7 @@ DevApps = new Mongo.Collection("devapps");
 //
 // Each contains:
 //   _id:  The application ID string (as with Packages.appId).
+//   appId: The same as _id, included to mimic Packages.appId.
```

Comment:
I'm OK with renaming the variable. You may want to leave the collection name unchanged in Mongo... though I guess all members of this collection are ephemeral so maybe it doesn't matter, but I worry that there's some problem I'm not thinking of with changing the collection name.